e2e/stack_test.go

```diff
@@ -317,52 +186,70 @@ func ethRollupFlow(t *testing.T, stack *e2e.StackConfig) {
 	// check that the user's balance has been updated on L1
 	require.Equal(t, expectedBalance, balanceAfterDeposit)
 
-	userCosmosAddr, err := userCosmosETHAddress.Encode("cosmos")
+	userCosmosAddr, err := userCosmosETHAddress.Encode("e2e")
 	require.NoError(t, err)
-	depositValueHex := hexutil.EncodeBig(depositAmount)
-	requireEthIsMinted(t, stack, userCosmosAddr, depositValueHex)
+	requireEthIsMinted(t, stack, userCosmosAddr, hexutil.EncodeBig(depositAmount))
 
 	t.Log("Monomer can ingest user deposit txs from L1 and mint ETH on L2")
 
 	/////////////////////////////
 	////// ETH WITHDRAWALS //////
 	/////////////////////////////
 
+	withdrawalAmount := big.NewInt(1)
+
 	// create a withdrawal tx to withdraw the deposited amount from L2 back to L1
-	withdrawalTx := e2e.NewWithdrawalTx(0, common.Address(userCosmosETHAddress), userETHAddress, depositAmount, new(big.Int).SetUint64(params.TxGas))
+	withdrawalTx := e2e.NewWithdrawalTx(0, common.Address(userCosmosETHAddress), userETHAddress, withdrawalAmount, new(big.Int).SetUint64(params.TxGas))
 
-	// initiate the withdrawal of the deposited amount on L2
-	senderAddr, err := monomer.CosmosETHAddress(*withdrawalTx.Sender).Encode("cosmos")
+	queryAccountBytes, err := protov1.Marshal(&authv1beta1.QueryAccountRequest{
+		Address: userCosmosAddr,
+	})
+	require.NoError(t, err)
+	result, err := stack.L2Client.ABCIQuery(stack.Ctx, authv1beta1.Query_Account_FullMethodName, queryAccountBytes)
```

Comment:
nit: `result` may seems like a pretty generic var name, could you rename it to something like `queryResult` for readability?